articles/article.re

```diff
@@ -0,0 +1,4 @@
+
+= 章タイトル
+
+ここからすべてが始まる！
```

Comment:
**[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Disallow to use "！". (ja-technical-writing/no-exclamation-question-mark)